packages/web3-providers/src/opensea/index.ts

```diff
@@ -433,4 +435,28 @@ export class OpenSeaAPI implements NonFungibleTokenAPI.Provider<ChainId, SchemaT
             collections.length === size ? createNextIndicator(indicator) : undefined,
         )
     }
+
+    async getCollectionStats(
+        address: string,
+        { chainId = ChainId.Mainnet }: HubOptions<ChainId> = {},
+    ): Promise<OpenSeaCollectionStats | undefined> {
+        const assetContract = await fetchFromOpenSea<OpenSeaAssetContract>(
+            urlcat('/api/v1/asset_contract/:address', { address }),
+            chainId,
+        )
+        const slug = assetContract?.collection.slug
+        if (!slug) return
+        const response = await fetchFromOpenSea<{ stats: OpenSeaCollectionStats }>(
+            urlcat('/api/v1/collection/:slug/stats', {
+                slug,
+            }),
+            chainId,
+            this._apiKey,
```

Comment:
We should not depend on a front-end API key anymore.

Reply:
Would be fixed by @Lanttcat